calendar_resource/README.rst

```diff
@@ -10,11 +10,26 @@ This module adds new features to facilitate resource management with meetings:
 
 * To attach resources to a meeting, go to `Calendar` => Click on a meeting => `Edit` =>
   `Edit` => add any related resources.
-* To edit or add a resource, go to `Settings` => `Resource` => `Resources`.
+* To edit or add a resource, go to `Settings` => `Resource` => `Resources`. Developer mode
+  has to be enabled for this menu to show.
```

Comment:
Aight so the plan is to create a new header in the top bar called `Resources` right? There are resource views in technical settings, so I need to move those to the new `Resources` menu.

That's what I'm getting at least. But I'm not sure whether to leave the resources menus in technical settings as-is or remove them.